### ISSUE TITLE
can't run the tensorflow/models/image/cifar10_train.py

### ISSUE THREAD

**Issue by 971456267** [tensorflow/tensorflow#8432]
2017-03-15 20:25:07.637177: step 0, loss = 4.68 (15.8 examples/sec; 8.118 sec/batch)
2017-03-15 20:25:08.461579: step 10, loss = 4.64 (1899.2 examples/sec; 0.067 sec/batch)
2017-03-15 20:25:09.127436: step 20, loss = 4.55 (1772.0 examples/sec; 0.072 sec/batch)
2017-03-15 20:25:09.786581: step 30, loss = 4.43 (2158.3 examples/sec; 0.059 sec/batch)
2017-03-15 20:25:10.453524: step 40, loss = 4.36 (2082.0 examples/sec; 0.061 sec/batch)
2017-03-15 20:25:11.129617: step 50, loss = 4.30 (1850.5 examples/sec; 0.069 sec/batch)
2017-03-15 20:25:11.791486: step 60, loss = 4.21 (2012.2 examples/sec; 0.064 sec/batch)
2017-03-15 20:25:12.466547: step 70, loss = 4.16 (1905.1 examples/sec; 0.067 sec/batch)
2017-03-15 20:25:13.120901: step 80, loss = 4.17 (1898.9 examples/sec; 0.067 sec/batch)
2017-03-15 20:25:13.770472: step 90, loss = 4.14 (2087.0 examples/sec; 0.061 sec/batch)
/home/fly/anaconda2/lib/python2.7/site-packages/IPython/core/interactiveshell.py:2889: UserWarning: To exit: use 'exit', 'quit', or Ctrl-D.
warn("To exit: use 'exit', 'quit', or Ctrl-D.", stacklevel=1)
An exception has occurred, use %tb to see the full traceback.
SystemExit
the information above was my problem.
when i run the code and set the max_steps = 100,the screen print "SystemExit" at step 90,when i set the code and set the max_steps = 1000,the screen print "SystemExit" at step 990.

**Comment by MCCCSunny** [comment]
It is a normal problem?
